PrebidMobile/PrebidMobileRendering/PluginRenderer/PluginEventListener.swift

```diff
@@ -0,0 +1,19 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
```

Comment:
<img width="776" alt="Screenshot 2024-05-16 at 10 51 37" src="https://github.com/prebid/prebid-mobile-ios/assets/99195905/0eb0c797-79c3-42a4-b4cd-6617c67ec2ae">

This class supposed to require for those implementing it the following field `pluginName: String`